src/panels/config/automation/ha-automation-picker.ts

```diff
@@ -898,6 +906,12 @@ class HaAutomationPicker extends SubscribeMixin(LitElement) {
     `;
   }
 
+  private _handleMenuClose(ev) {
```

Comment:
1. Add a type for `ev` so it's not `any` (should be importable I would think)
2. Define this handler on the `HaMenu` class.  There's nothing instance -specific here.

Reply:
I'm not quite sure what to type this. Am I just supposed to copy and paste this from their documentation, or is there something more succinct?

`CustomEvent<{initiator: SelectOption, reason: Reason, itemPath: SelectOption[]}>`